tests/integration/static_ip_address_test.py

```diff
@@ -87,8 +87,7 @@ def setup_eth1_ipv6(eth1_up):
         ]
     }
     libnmstate.apply(desired_state)
-
-    return desired_state
+    yield desired_state
```

Comment:
Why is this needed? If the function is not doing anything after the `yield` there is no difference between `yield` and `return` AFAIK. Sorry if I am missing something, thanks!

Reply:
Just for consistent, other fixture in test codes use `yield`.
And I don't know whether `return` still run the clean up of `eth1_up`, and there is no document said so, but for `yield`, it does.

Reply:
You could argue the same for a regular function, but then it will make it a generator.
Yielding implies you have a teardown to the fixture while return explicitly say there is none.

I would argue it is better to use each per the need to help the reader understand the intention clearly.

Regarding correctness, it is like returning from a contextmanager, the upper context/fixture teardown will kick in.